src/components/Table/Table/Table.tsx

```diff
@@ -43,35 +45,30 @@ interface ITableContext {
 
 export const TableContext = React.createContext<ITableContext>(null);
 
-const Table: FC<ITableProps> = ({
-  id,
-  className,
-  "data-testid": dataTestId,
-  columns,
-  errorState,
-  emptyState,
-  dataState,
-  style,
-  children
-}) => {
-  const classNames = cx(styles.table, className);
-  const { gridTemplateColumns } = getTableRowLayoutStyles(columns);
+const Table: VibeComponent<ITableProps> = forwardRef(
+  ({ id, className, "data-testid": dataTestId, columns, errorState, emptyState, dataState, style, children }, ref) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
```

Comment:
Why we need componentRef and mergedRef? Why just not to pass ref directly? 
Same comment in all of the components